app/javascript/components/Pages/Borrow.js

```diff
@@ -1,11 +1,34 @@
 import React, { Component } from "react"
+import { Link } from 'react-router-dom';
+import { Button } from 'reactstrap'
 
-export default class Borrow extends Component {
-    render(){
-        return(
-            <>
-                <h3>Borrow</h3>
-            </>
-        )
-    }
-}
\ No newline at end of file
+const Borrow = ({ books }) => (
+
+    <>
+        <h3 className="heading">All Books</h3>
+        { books.map((book, index) => {
+            return (
+                <div key={index} className="bookCard">
```

Comment:
This totally works, but as apps grow, it is easy to start having duplicate keys.  React uses these keys to identify the component and to optimize re-renders when [reconciling](https://reactjs.org/docs/faq-internals.html).  So, in this case I would recommend something like: ``key={`book-${index}`}``.

Reply:
Very interesting! We will definitely edit this stuff and push back up tomorrow. Thank you!